lib/ast-node-types.js

```diff
@@ -140,17 +139,16 @@ module.exports = {
     TSStaticKeyword: "TSStaticKeyword",
     TSStringKeyword: "TSStringKeyword",
     TSSymbolKeyword: "TSSymbolKeyword",
+    TSTypeAnnotation: "TSTypeAnnotation",
     TSTypeLiteral: "TSTypeLiteral",
+    TSTypeParameter: "TSTypeParameter",
```

Comment:
Looks like this will conflict with `TypeParameter`s that were already being copied over :(

Example with mapped types:

`var x: { [A in keyof B]?: any };`

Ref: https://github.com/prettier/prettier/pull/1393

Reply:
Thanks a lot for your patience on this @existentialism, I have a fix for that in prettier and have opened a PR here: https://github.com/prettier/prettier/pull/3455